kbcstorage/workspaces.py

```diff
@@ -0,0 +1,112 @@
+"""
+Manages workspace requests to the API.
+
+Full documentation `here`.
+
+.. _here:
+    http://docs.keboola.apiary.io/#reference/workspaces/
+"""
+
+from kbcstorage.base import Endpoint
+
+
+def _make_body(mapping):
+    """
+    Given a dict mapping Keboola tables to aliases, construct the body of
+    the HTTP request to load said tables.
+
+    Args:
+        mapping(:obj:`dict`): Keys contain the full names of the tables to
+            be loaded (ie. 'in.c-bucker.table_name') and values contain the
+            aliases to which they will be loaded (ie. 'table_name').
```

Comment:
table mapping has other parameters like `days`, `rows`, etc. while it is not important to support them now, the structure has to be extensible, i.e. i'd say it either has to be array of dicts (each wit keys source, destination, etc) or a designated object